packages/lib-classifier/src/components/Classifier/components/QuickTalk/QuickTalk.spec.js

```diff
@@ -69,23 +68,24 @@ describe('Component > QuickTalk', function () {
         />
       )
     })
-    
+
     it('should render without crashing', function () {
-      expect(wrapper.queryByTestId('quicktalk-button')).to.exist()
-      expect(wrapper.queryByTestId('quicktalk-panel')).to.not.exist()
+      expect(screen.queryByTestId('quicktalk-button')).to.exist()
+      expect(screen.queryByTestId('quicktalk-panel')).to.not.exist()
     })
-    
-    it('should expand when clicked', function () {
-      fireEvent.click(wrapper.queryByTestId('quicktalk-button'))
-      
-      expect(wrapper.queryByTestId('quicktalk-button')).to.not.exist()
-      expect(wrapper.queryByTestId('quicktalk-panel')).to.exist()
+
+    it('should expand when clicked', async function () {
+      const user = userEvent.setup({ delay: null })
+      await user.click(screen.queryByTestId('quicktalk-button'))
```

Comment:
```js
screen.queryByRole(‘button’, { name: ‘Close comments panel’ })
```

Reply:
Getting buttons by role and label seems better than adding IDs.

Reply:
Thanks Jim, I've updated these button queries with roles. I've also added role=dialog to the quick talk panel